inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -39,27 +52,38 @@ void LayerTestsCommon::Run() {
     s.setDeviceName(targetDevice);
 
     if (FuncTestUtils::SkipTestsConfig::currentTestIsDisabled()) {
-        s.updateOPsStats(function, PassRate::Statuses::SKIPPED);
+        s.updateOPsStats(functionRefs, PassRate::Statuses::SKIPPED);
         GTEST_SKIP() << "Disabled test due to configuration" << std::endl;
     } else {
-        s.updateOPsStats(function, PassRate::Statuses::CRASHED);
+        s.updateOPsStats(functionRefs, PassRate::Statuses::CRASHED);
     }
 
     try {
         LoadNetwork();
-        GenerateInputs();
-        Infer();
-        Validate();
-        s.updateOPsStats(function, PassRate::Statuses::PASSED);
+        for (size_t i = 0; i < targetStaticShapes.size(); i++) {
```

Comment:
I see that some tests have empty 'targetStaticShapes'. Does it mean that these tests will now silently 'pass' without verifying anything?

Reply:
Fixed: https://github.com/openvinotoolkit/openvino/pull/7829